deploy/stand-up-dev-env.sh

```diff
@@ -18,7 +18,7 @@ DEPLOY_PATH=${STRATOS_UI_PATH}/deploy
 
 NO_UI=false
 CLEAN=false
-
+STRATOS_VERSION=v2.0.0-$(git log -1 --format="%h")
```

Comment:
Can we remove the v  so its 2.0.0 ... We should read the 2.0.0 part from the package file, otherwise we'll need to update this everytime we release 2.0.1 etc.